code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -73,10 +75,20 @@ func (edb *EventDb) addEventsWorker(ctx context.Context) {
 			case TypeStats:
 				err = edb.addStat(event)
 			case TypeError:
-				err = edb.addError(Error{
-					TransactionID: event.TxHash,
-					Error:         event.Data,
-				})
+
+				msg, ok := fromEvent[string](event.Data)
```

Comment:
We don't need to use generics to get the data, `fmt.Sprintf("%v", event.Data)` is simpler and should have  the same effort.

Reply:
fixed